ban/core/versioning.py

```diff
@@ -272,6 +272,13 @@ def serialize(self, *args):
 
 class Redirect(db.Model):
 
+    __openapi__ = """
+        properties:
+            identifier_value:
```

Comment:
What we call an "identifier" usually is already a "key:value" pair (or the special case of the id itself, if the key is missing), so I think we can just call it "identifier" here.
Just make clear in the description that it's a "key:value" pair.